analyzers/src/SonarAnalyzer.Common/Common/AnalyzerConfiguration.cs

```diff
@@ -84,20 +85,20 @@ public void Initialize(AnalyzerOptions options)
 
             public void Initialize(AnalyzerOptions options)
             {
-                var currentSonarLintXmlPath = GetSonarLintXmlPath(options);
-                if (isInitialized && loadedSonarLintXmlPath == currentSonarLintXmlPath)
+                var currentSonarLintXml = GetSonarLintXml(options);
```

Comment:
The old name `currentSonarLintXmlPath` made sense because it was clear it referred to a path. 
However, I found `currentSonarLintXml` a bit confusing because it suggest the variable refers to XML.

It's difficult to think of a good name that isn't too long. `currentSonarLintXmlFile` maybe, since the method is returning an additional file?

I'd rename the method to `GetSonarLintXmlFile` too.